.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: ❓ Questions
+    url: https://github.com/microsoft/torchgeo/discussions
```

Comment:
Could replace this with a Slack invite link if we ever decide to go down that route